open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -13,10 +13,25 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ax=df[df.index&lt;"2020-06-01"]['ninfected'].diff().rolling(7).mean().plot()
</pre><p>You can use the <code>get_diff</code> and <code>get_rolling_window_mean()</code>  defined  above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>Sorry, combine failed..</p>